bokeh/models/tests/test_annotations.py

```diff
@@ -340,6 +345,9 @@ def test_Span():
         "render_mode"
     ], LINE)
 
+def test_Span_accepts_datetime_location():
+    obj = Span(location = pd.to_datetime('2018-8-7'))
+    assert obj.location == 1533600000000.0
 
 def test_Title():
     title = Title()
```

Comment:
@jasonsross one small change is needed. We have a test that makes sure that basic things work without pandas installed. So the test should not import pandas directly. There are two options:

* add an argument `pd` to the two test functions, e.g `def test_Label_accepts_datetime_location(pd)`. We have a pytest fixture that automatically will provide pandas if it installed or skip the test if not

* use the built in `datetime` module, or `numpy` datetime functions, instead of pandas

Either is ok with me!

Reply:
@bryevdv sounds good I've dropped pandas for datetime so if pandas is not installed, we don't lose this test.